packages/client/hmi-client/src/page/project/components/tera-project-overview.vue

```diff
@@ -364,12 +364,13 @@ async function processFiles(files: File[], csvDescription: string) {
 			case AcceptedTypes.PDF:
 			case AcceptedTypes.TXT:
```

Comment:
Should .txt files still be considered artifacts? 
The document viewer is not designed or .txt

![image](https://github.com/DARPA-ASKEM/terarium/assets/17088680/984023e4-48ca-41b2-b980-6a883276b3d2)